index.html

```diff
@@ -94,90 +94,13 @@ <h1 class="greeting-text">Hey there. I’m <br> Abdul Ali.</h1>
 <!-- ----------------------------------------Recent work Setion------------------------------------------- -->
 
             <section class="recent-work grid" id="recent-work">
+                
```

Comment:
- [ ] Please make sure to fix the buttons they should look exactly look like the design in all sizes mobile and desktop also fix the distance between them they should have a distance as in the design.
your design 
![image](https://user-images.githubusercontent.com/74744787/217950525-955173f1-67cb-4164-8ab6-81eaedef4b0d.png)
Figmas 
![image](https://user-images.githubusercontent.com/74744787/217950647-4b566b30-edeb-4673-ac69-f0710d43b75f.png)